config/locales/pt-BR.yml

```diff
@@ -0,0 +1,47 @@
+pt-BR:
+  helpers:
+    submit:
+      create: Novo
```

Comment:
I don't think we should add any translations outside of the godmin scope. It will affect the entire site. If we want these to be translatable only in godmin we should add specific translations for them inside the godmin scope.

Can you revert these changes and squash it into one commit please?